targets/simple_switch/externs/sym_crypto/README.md

```diff
@@ -0,0 +1,22 @@
+# ADD CRYPTO to SIMPLE_SWITCH
+
+For reference, the extern is implemented in crypto.cpp in this directory.
```

Comment:
There are no instructions here on how to compile the attached P4 program using p4c.

Reply:
I can add text.  In the macsec repo this Makefile is used to compile with p4c.

https://github.com/uni-tue-kn/p4-macsec/blob/master/p4/p4/Makefile